postgresql-12/postgresql-entrypoint.sh

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env bash
```

Comment:
Hard to tell at a glance how different these files are, and if there might be some way to share them. Is there any benefit to having one `postgresql-entrypoint.sh` that gets referenced by each pipeline?

Reply:
Yeah they're exactly the same - I did wonder whether to reference a single one or duplicated them for each version as the melange source directory mounts a folder for sources, that matches the package name.  In this postgres case it means a folder name with the version number in it.  

Having one is better though, will switch and have a hard link to the `postgresql-entrypoint.sh`